tools/git/commit-msg.sh

```diff
@@ -15,8 +15,8 @@
 # limitations under the License.
 
 # Check commit message: https://www.conventionalcommits.org/en/v1.0.0-beta.2/
-if ! which commitlint; then
+if ! which commitlint &>/dev/null; then
   echo "FAILURE: commitlint not installed"
```

Comment:
Sorry for the late comment, but this should probably go to stderr: `>&2`

Reply:
done